templates/zerver/integrations/jenkins.md

```diff
@@ -27,32 +29,47 @@ Jenkins.)
 ### Configure a post-build action for your project
 
 Once you've done that, it's time to configure one of your
-projects to use the Zulip notification plugin. From your
+freestyle projects to use the Zulip notification plugin. From your
```

Comment:
`On your Jenkins project page` is better here probably. At every point in the docs, we want to be careful about whether we are referring to the Zulip UI or the third-party UI. :)